source/MonoGame.Extended/FNA.Extended.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>It makes FNA more awesome.</Description>
+    <PackageTags>fna extended pipeline bmfont tiled texture atlas input viewport fps shapes sprite</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Remove="MonoGame.Framework.DesktopGL" />
+
+    <ProjectReference Include="..\..\..\FNA\FNA.NetStandard.csproj">
```

Comment:
FNA provides different projects for net4.0, NetStandard2.0, Net8.
Building against the NetStandard is the only option compatible with our current .net6 target.